### ISSUE TITLE
Sorting clears filters in Org search

### ISSUE THREAD

**Issue by mariongalley** [ThreeSixtyGiving/grantnav#895]
**Describe the bug**
If you have non-GBP currencies selected, then try to change the sort, the selections are removed. [Here is the link to test](http://grantnav-org-pages.360dev1.vs.mythic-beasts.com/recipients?query=%2A&default_field=%2A&sort=aggregate.grants+desc&currency=USD&currency=EUR&currency=CHF&currency=ILS&currency=CAD).

**To Reproduce**
1. Go to Funders or Recipients search
2. Select a Non GBP Currency
3. Sort by any of the sort options
4. Currency filter disappears

**Expected behavior**
Filters should 'stick' when sort options are changed. If the sort cannot work for non GBP currencies, it should fail more gracefully.

**Comment by michaelwood** [comment]
Fixed deployed to dev. The behaviour when trying to sort on GBP when someone has filtered by USD is now just ignored.

**Comment by mariongalley** [comment]
Not sure ignoring completely is ideal in the long term. What are the options @michaelwood - either present an error temporarily saying you can't sort by GBP or conditionally hide the GBP sort options when any non-GBP currency is selected?

@KDuerden Are we happy to call this done for MVP but perhaps raise a separate ticket to make it provide some kind of feedback?

**Comment by michaelwood** [comment]
> Not sure ignoring completely is ideal in the long term. What are the options @michaelwood - either present an error temporarily saying you can't sort by GBP or conditionally hide the GBP sort options when any non-GBP currency is selected?
> 

Yes, all of those things are possible, as well as just removing the sort all together.

> @KDuerden Are we happy to call this done for MVP but perhaps raise a separate ticket to make it provide some kind of feedback?

Yes please do open another issue to track a design change.

**Comment by mariongalley** [comment]
@KDuerden It seems that the GBP sorting does actually work when you've filtered to another currency, it just sorts the highest/lowest GBP amount for funders that have awarded in GBP as well as the other currency(ies) selected
